practice-app/back-end/routes/searchtrendforproduct.js

```diff
@@ -0,0 +1,59 @@
+
+var express = require('express');
+var router = express.Router();
+
+
+router.get('/', function(req, res, next) {
+
+  //create a Twitter object with tokens
+  var Twitter = require('twitter');
+  var key = require('./keys.js');
+  var Twit = new Twitter(key);
+
+  var today = new Date();
+  var date = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate(); //date of today
+  var yesterday = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+(today.getDate()-1); //date of yesterday
```

Comment:
does that work in 1st day of months? 
or first january?

Reply:
sorry couldn't understand. It works for the current day

Reply:
I think what @kayacanv is trying to say is that if we currently are in the first day of month and subtract 1 from current day to get yesterday can be a problem like 1-1=0 and we don't have 0 day.

Reply:
I made small changes in the parameters so there is no need for yesterday variable anymore.